extension/package.json

```diff
@@ -664,11 +664,11 @@
         },
         {
           "command": "dvc.applyExperiment",
-          "when": "dvc.commands.available && dvc.project.available && !dvc.experiment.running"
+          "when": "dvc.commands.available && dvc.project.available && !dvc.experiment.running.workspace"
```

Comment:
[F] We only want to suppress actions when there is an experiment running in the workspace.